### ISSUE TITLE
NPM packages without repository urls

### ISSUE THREAD

**Issue by rhazarian** [webjars/webjars#2052]
Hello!

I have another dependency of `graphiql` that doesn't contain a repository url in its package configuration: https://www.npmjs.com/package/@motionone/dom. The version of `@motionone/dom` that is needed by the `graphiql` version I'm currently trying to deploy is `10.12.0`. It seems that here is their original repository: https://github.com/motiondivision/motionone/tree/main/packages/dom. The expected arifactId is `motionone__dom`.

Do you think it would be possible to modify WebJars deployment mechanism so it could grab the files directly from NPM? If it's possible, I believe it would cover all of the possible cases and free you from these manual deployment requests. Honestly, I feel bad for bothering you already :), and I'm afraid that this may not be the last NPM package without a repo url that is present in this huge dependency graph. I encounter a lot of popular NPM packages without repository urls specified.

**Comment by jamesward** [comment]
Yeah, this is a pain. Back story is that we used to deploy via Bintray and they required a source repo. But Maven Central doesn't have that requirement so we could remove the requirement.  It'll just take some work to do that.  Until then it isn't hard to manually deploy these repos.  I've deployed this:
```
GroupID = org.webjars.npm
ArtifactID = motionone__dom
Version = 10.12.0
```

I'll keep this issue open as a reminder to remove the requirement.

**Comment by rhazarian** [comment]
Makes sense, thanks for the explanation!

It seems that there are some more libraries from the same 'set' that are needed. `@motionone/animation` (https://www.npmjs.com/package/@motionone/animation), `@motionone/generators` (https://www.npmjs.com/package/@motionone/generators), `@motionone/types` (https://www.npmjs.com/package/@motionone/types), `@motionone/utils` (https://www.npmjs.com/package/@motionone/utils). Their sources are located here: https://github.com/motiondivision/motionone/tree/main/packages. The needed version is `10.12.0`.

Thank you for your help, I really hope that we are on the finish line! :)

**Comment by jamesward** [comment]
Those have all been deployed and should be in Maven Central shortly.

**Comment by rhazarian** [comment]
Apparently, I've skipped one of them :(. There is also `@motionone/easing` (https://www.npmjs.com/package/@motionone/easing). Similarly, the required version is `10.12.0` - it's a dependency of the previously deployed `@motionone/animation`.

**Comment by jamesward** [comment]
No problem at all! That has been deployed:
```
GroupID = org.webjars.npm
ArtifactID = motionone__easing
Version = 10.12.0
```
